programs/range/src/main.rs

```diff
@@ -49,6 +49,7 @@ fn main() {
             .await
             .expect("failed to run client");
 
+        sp1_zkvm::io::commit(&BootInfoStruct::from(boot_info.clone()));
```

Comment:
The `range` program should not be modified?

Reply:
I intentionally made a change to the program to see if the CI works. Will drop the commit before merging.